pkg/tinydtls/contrib/sock_dtls.c

```diff
@@ -259,6 +259,12 @@ int sock_dtls_create(sock_dtls_t *sock, sock_udp_t *udp_sock,
     return 0;
 }
 
+sock_udp_t *sock_dtls_get_udp_sock(sock_dtls_t *sock)
+{
+    assert(sock);
+    return sock->udp_sock;
```

Comment:
Why not `static inline`?

Reply:
Nope, this will lead to compile errors, ones the function is used by multiple objects. This is a public function of the `sock_dtls` interface accessing a member module-visible only to the `tinydtls` implementation of said interface. It's the same as e.g. `sock_udp_get_local()` ;-)

Reply:
Also if, for whatever reason an implementation needs to do more complicated things than just referencing a member, it would be screwed.

Reply:
Well of course I meant that then should also be in the header then 😉 

But yea if the contents of `sock_dtls_t` are private, then that’s the way to go.

Reply:
> But yea if the contents of `sock_dtls_t` are private, then that’s the way to go.

As with all `sock` types, they are, yes :-).